File: _01_Languages/Kotlin/src/_01_basicos/_04_Condicionales.kt
/*
 * 🧩 Enunciado:
 * Pide la edad del usuario y muestra:
 * - "Menor de edad" si es <18
 * - "Adulto" si está entre 18 y 65
 * - "Jubilado" si es >=65
 */

fun main() {
    print("Introduce tu edad: ")
    val edad = readLine()!!.toInt()

    when {
        edad < 18 -> println("👶 Eres menor de edad.")
        edad < 65 -> println("🧑 Eres adulto.")
        else -> println("👴 Eres jubilado.")
    }
}

File: _01_Languages/CSharp/src/_01_basicos/_04_Condicionales.cs
/*
 * 🧩 Enunciado:
 * Pide al usuario su edad y muestra:
 * - "Menor de edad" si es < 18
 * - "Adulto" si está entre 18 y 65
 * - "Jubilado" si es >= 65
 */

using System;

class _04_Condicionales
{
    static void Main()
    {
        Console.Write("Introduce tu edad: ");
        int edad = Convert.ToInt32(Console.ReadLine());

        if (edad < 18)
            Console.WriteLine("👶 Eres menor de edad.");
        else if (edad < 65)
            Console.WriteLine("🧑 Eres adulto.");
        else
            Console.WriteLine("👴 Eres jubilado.");
    }
}
